src/commands/Tools/eshop.ts

```diff
@@ -0,0 +1,153 @@
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage, Timestamp, util } from 'klasa';
+import { stringify } from 'querystring';
+import { TOKENS } from '../../../config';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserRichDisplay } from '../../lib/structures/UserRichDisplay';
+import { cutText, fetch, getColor } from '../../lib/util/util';
+import { decode } from 'he';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_ESHOP_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_ESHOP_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			usage: '<gameName:string>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [gameName]: [string]) {
+		const response = await message.sendEmbed(new MessageEmbed()
+			.setDescription(message.language.tget('SYSTEM_LOADING'))
+			.setColor(getColor(message) || 0xFFAB2D));
+
+		const { results: entries } = await fetch(`https://${TOKENS.NINTENDO.ID}-dsn.algolia.net/1/indexes/*/queries`, {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+				'X-Algolia-API-Key': TOKENS.NINTENDO.KEY,
+				'X-Algolia-Application-Id': TOKENS.NINTENDO.ID
+			},
+			body: JSON.stringify(
+				{
+					requests: [
+						{
+							indexName: 'noa_aem_game_en_us',
+							params: stringify({
+								facetFilters: [
+									'type:game'
+								],
+								hitsPerPage: 10,
+								maxValuesPerFacet: 30,
+								page: 0,
+								query: gameName
+							})
+						}
+					]
+				}
+			)
+		}, 'json')
+			.catch(() => { throw message.language.tget('SYSTEM_QUERY_FAIL'); }) as EshopResult;
+
+		const display = this.buildDisplay(entries[0].hits, message);
+
+		await display.start(response, message.author.id);
+		return response;
+	}
+
+	private buildDisplay(entries: EShopHit[], message: KlasaMessage) {
+		const display = new UserRichDisplay();
+
+		for (const game of entries) {
+			const titles = message.language.language.COMMAND_ESHOP_TITLES as unknown as EshopLanguage;
+			const description = decode(cutText(game.description.replace(/\s\n {2,}/g, ' '), 750));
+			const esrbText = game.esrb
+				? [
+					`**${game.esrb}**`,
+					game.esrbDescriptors && game.esrbDescriptors.length ? ` - ${game.esrbDescriptors.join(', ')}` : ''
+				].join('')
+				: message.language.tget('COMMAND_ESHOP_NOT_IN_DATABASE');
+			let price = 'Free';
+			if (game.msrp && game.msrp > 0) price = `$${game.msrp} USD`;
```

Comment:
```suggestion
			const price = game.msrp > 0 ? `$${game.msrp} USD` : 'Free';
```

But also move this to i18n.